### PR TITLE
add ssr

### PR DESCRIPTION
**Pull request by therealmarv**
ssr, wip...

### CODE REVIEW

File: workflow-service/frontend/nginx.conf [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-    try_files $uri $uri/ /index.html =404;
-  }
+        proxy_redirect                      off;
+        proxy_set_header Host               $host;
+        proxy_set_header X-Real-IP          $remote_addr;
+        proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto  $scheme;
+        proxy_read_timeout          1m;
+        proxy_connect_timeout       1m;
+        proxy_pass                          http://127.0.0.1:3000; # set the adress of the Node.js instance here
```
Comment (m1yag1) [inline workflow-service/frontend/nginx.conf]: address typo ;)

File: workflow-service/frontend/Dockerfile [hunk 2]
```diff
@@ -12,11 +15,10 @@ ARG FRONTEND_ENV=production
 
 ENV VUE_APP_ENV=${FRONTEND_ENV}
```
Comment (m1yag1) [inline workflow-service/frontend/Dockerfile]: What's the difference between production and development?

File: workflow-service/frontend/nginx.conf (hunk 1 above)
Comment (therealmarv) [reply workflow-service/frontend/nginx.conf]: haha, that's an upstream bug. The original nuxt authors are French and they say "l'adress". I created a PR for it https://github.com/nuxt/docs/pull/1688 😁

File: workflow-service/frontend/Dockerfile (hunk 2 above)
Comment (m1yag1) [reply workflow-service/frontend/Dockerfile]: Does it cause differences in the behavior. I'm just wondering.

Comment (therealmarv) [reply workflow-service/frontend/Dockerfile]: no, not now. Maybe we can remove it. I'm doing a local test currently with setting base url to `/` for production and docker setups.

**Review by m1yag1 — APPROVED** [review]

**Pull request merged**